examples/lisa/run_inference.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Add script to download the data files for the test example. Don't commit large data files though.